libs/shared/console-shared/src/lib/flow-create-port/ui/crud-modal/crud-modal.tsx

```diff
@@ -260,6 +262,52 @@ export function CrudModal({
           <Callout.Text>Please verify the health check configuration.</Callout.Text>
         </Callout.Root>
       )}
+      {isEdit && !watchPublicly && getFieldState('publicly_accessible').isDirty && isLastPublicPort && (
+        <div>
+          <Callout.Root className="mt-5" color="red">
+            <Callout.Icon>
+              <Icon iconName="triangle-exclamation" className="text-red-600" />
+            </Callout.Icon>
+            <Callout.Text>
+              <div className="flex flex-col gap-2">
+                <p className="font-semibold">
+                  You are about to remove your last public port.
+                  <br />
+                  Please confirm that you understand the impact of this operation.
+                </p>
+                <div className="flex items-center gap-4">
+                  <Controller
+                    name="confirm"
+                    control={control}
+                    rules={{
+                      required: 'Please check the box to confirm.',
+                      validate: (value) => value,
```

Comment:
Is it really necessary ? You've the same thing inside the `modal-multi-confirmation`